### DISCUSSION TITLE
set-cookie, on and off. allow and disallow

### DISCUSSION THREAD

**Discussion by ryanbr** [uBlockOrigin/uAssets#20350] [General]
From "mypensiontracker.co.uk"

mypensiontracker.co.uk##+js(set-cookie, complianceCookie, on)  
! praxisplan.at
praxisplan.at##+js(trusted-set-cookie, trackingCookies, off)
praxisplan.at##+js(trusted-set-cookie, complianceCookie, on)

`on, off`

teket.jp##+js(trusted-set-cookie, _ul_cookie_consent, allow)

`allow, disallow`
maybe; `allowed, disallowed`

**Comment by ryanbr** [comment]
Optional set-cookie also; 

```
essential
required
optional
functional
necessary
marketing
disabled / disable
enabled / enable
```

**Comment by ryanbr** [comment]
`crown.com##+js(set-cookie, modalPolicyCookieAccepted, accepted)`

`notaccepted`, `accepted`, 

Maybe also: `reject`, `rejected`

**Comment by ghost** [comment]
`on`/`off` already added https://github.com/gorhill/uBlock/blame/3c4932e1a68fd8a0ca14a1932b4b7e7c4164d195/assets/resources/scriptlets.js#L3381C21-L3381C21

**Comment by ghost** [comment]
You can create PR like this one https://github.com/gorhill/uBlock/pull/3898

**Reply by ryanbr** [reply]
I could, though I don't want to go through the process just to be rejected

**Reply by ghost** [reply]
It will be easier to accept when it will be ready, just one click away :)

Listing all the real-life use cases will make it even more tempting :)

**Reply by gorhill** [reply]
I need to see real cases for each of the suggested addition, otherwise there is no end to what we might think _could_ be useful.
